Example/Tests/Tests.m

```diff
@@ -63,6 +63,15 @@
 
         [verify(mixpanel) track:@"Viewed Home Screen" properties:@{}];
     });
+    
+    it(@"screen with consolidatedPageCalls", ^{
+        [integration setSettings:@{
+            @"consolidatedPageCalls" : @true
```

Comment:
Since we extract the setting as a `NSNumber` first, let's use a number here as well since that's closer to what goes on in a production environment.